app/javascript/__tests__/modules/listingDetails/ListingDetailsPreferences.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import renderer, { act } from "react-test-renderer"
+
+import { ListingDetailsPreferences } from "../../../modules/listingDetails/ListingDetailsPreferences"
+import { preferences as defaultPreferences } from "../../data/lottery-preferences-default"
+import { preferences as sixPreferences } from "../../data/lottery-preferences-six"
+
+import { getPreferences } from "../../../api/listingApiService"
+
+jest.mock("../../../api/listingApiService")
+
+describe("ListingDetailsPreferences", () => {
+  it("display 3 default preferences - COP, DHTP, L/W", async () => {
+    const getPreferencesMock = getPreferences as jest.MockedFunction<typeof getPreferences>
+    getPreferencesMock.mockReturnValue(Promise.resolve(defaultPreferences))
+
+    const tree = renderer.create(<ListingDetailsPreferences listingID={"test"} />)
+
+    // wait for state changes
+    await act(() => new Promise((resolve) => setTimeout(resolve)))
+
+    expect(tree.toJSON()).toMatchSnapshot()
+  })
```

Comment:
was debating whether to add in [msw](https://mswjs.io/) but opted for this shorter path for now